build-system/compile/sources.js

```diff
@@ -41,6 +41,8 @@ const COMMON_GLOBS = [
   'third_party/webcomponentsjs/ShadowCSS.js',
   'node_modules/dompurify/dist/purify.es.js',
   'node_modules/promise-pjs/promise.mjs',
+  'node_modules/rrule/dist/esm/src/rrulestr.js',
+  'node_modules/rrule/dist/esm/src/parsestring.js',
```

Comment:
Nit: Just allow everything under esm folder?

```suggestion
  'node_modules/rrule/dist/esm/src/*.js',
```